### PR TITLE
Added usage scenarios

### PR DESCRIPTION
**Pull request by pramit-d**
https://github.com/Be-Secure/oah-bes-vm/issues/29

Added usage scenarios.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
-## What is oah-bes-vm 
+## What is oah-bes-vm?
 
 A substrate that brings up bundles of BeS & other tools to perform Red Team & Blue Team activity.
 
 ## Who are the end users?
 
 RT & BT activist of BeS community.
```
Comment (panickervinod) [inline README.md:7]: @pramit-d link to the definition of RT & BT , everyone need not know what RT & BT is

File: README.md [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 
+### Usage scenarios:
+1. **Security Analyst**: RT & BT activist of BeS community can use oah-bes-vm to work on a specific open-source security project.
```
Comment (panickervinod) [inline README.md:10]: Link to explanation of who a Security Analyst is in the context of this scenario

File: README.md [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
+
+2. **BeSLab Manager**: oah-bes-vm helps BeSLab Manager & Security Analyst to analyse and do RT & BT activities of open-source security projects. After analysing multiple open-source security projects using multiple static analysis tools (like SonarQube, Codeql, SBOM etc) by BeSLab Manager, Security Analyst can use oah-bes-vm to setup an isolated RT/BT environment for each spesific open-source security project to do RT & BT activities.
```
Comment (panickervinod) [inline README.md:12]: Link to where there is documentation on the what a BeSlab and the role of a BeSLab admin

File: README.md [hunk 4]
```diff
@@ -1,11 +1,15 @@
...
 ## Features of oah-bes-vm?
 
 1. Offers bundles of BeS tools that would enable users to fast track their Red Teaming / Blue Teaming activities by significantly reducing effort spent on setting up respective environments.
```
Comment (panickervinod) [inline README.md:15]: @pramit-d @asa1997 @sudhirverma does not look to the point to me.

File: README.md (hunk 1 above)
Comment (pramit-d) [reply README.md]: @panickervinod added RT BT definition.

File: README.md (hunk 4 above)
Comment (pramit-d) [reply README.md]: @panickervinod Updated readme file.

File: README.md (hunk 2 above)
Comment (pramit-d) [reply README.md:16]: Added keywords.md file inside docs, will link it with the Security Analyst keyword after the current PR is merged.

File: README.md [hunk 5]
```diff
@@ -1,20 +1,50 @@
-## What is oah-bes-vm 
+## What is oah-bes-vm?
 
-A substrate that brings up bundles of BeS & other tools to perform Red Team & Blue Team activity.
+A substrate that brings up bundles of BeS & other tools to perform Red Team & Blue Team activity. It offers bundles of BeS tools that would enable users to fast track their Red Teaming / Blue Teaming activities by significantly reducing effort spent on setting up respective environments.oah-bes-vm helps to isolate the installation of BeS projects in a specific environment. According to the needs, user can configure oah-bes-vm during installation.
```
Comment (sudhirverma) [inline README.md:3]: Can you provide space after 
```suggestion
A substrate that brings up bundles of BeS & other tools to perform Red Team & Blue Team activity. It offers bundles of BeS tools that would enable users to fast track their Red Teaming / Blue Teaming activities by significantly reducing effort spent on setting up respective environments. oah-bes-vm helps to isolate the installation of BeS projects in a specific environment. According to the needs, user can configure oah-bes-vm during installation.
```

File: README.md [hunk 6]
```diff
@@ -1,20 +1,50 @@
...
+
+   - ### Uninstall an environment
+            bes uninstall -env <environment name>
+     ![uninstall environment](docs/img/env_remove.gif)
+
+    For more information check [BeSman documentation](https://be-secure.github.io/Be-Secure/bes-besman-details/).
 ## List of tools
 ### BeS Tools
-1. BeSman
+1. [BeSman](https://be-secure.github.io/Be-Secure/bes-besman-details/)
```
Comment (sudhirverma) [inline README.md:47]: Can you give some overview on List of tools and why they are required in oah-bes-vm?

**Review by sudhirverma — APPROVED** [review]

**Pull request merged**